pygmt/accessors.py

```diff
@@ -11,15 +11,36 @@
 @xr.register_dataarray_accessor("gmt")
 class GMTDataArrayAccessor:
     """
-    GMT extension for :class:`xarray.DataArray`.
+    GMT accessor for :class:`xarray.DataArray`.
 
-    The extension provides easy ways to access and change the GMT specific
-    properties about grids. Currently, two properties are available:
+    The accessor extends :class:`xarray.DataArray` to store GMT-specific
+    properties about grids, which are important for PyGMT to correctly process
```

Comment:
Technically xarray accessors are not just about setting properties/attributes. You may need to change this sentence if the `.gmt.imshow()` function at #2372 is added to this accessor class.

Reply:
Will rephrase this sentence in PR #2372.